docker-integrations/tendermint-python/src/main.py

```diff
@@ -0,0 +1,31 @@
+import concurrent.futures
+import sys
+
+import grpc
+
+# Add the parent folder to the module search path
+parent_folder = "src/protos"
+sys.path.append(parent_folder)
+
+from protos import runtime_pb2_grpc
+from server import TendermintServer
+
+IP_ADDR = "runtime:50051"
```

Comment:
Also I would not call it `IP_ADDR` because this format `hostname:port` does not equal an IP address. If we want to be super technical it would be a socket, but we can more casually call it "endpoint".

Reply:
I rename the one in `common/docker` to be `RUNTIME_CLIENT_ADDR` and kept the same format in the runtime with `RUNTIME_SERVER_ADDR`